go.mod

```diff
@@ -969,3 +994,7 @@ exclude github.com/docker/distribution v2.8.0+incompatible
 
 // see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12934
 replace github.com/outcaste-io/ristretto => github.com/outcaste-io/ristretto v0.1.1-0.20220420003845-bd658a460d49
+
+// some dependencies attempt to bring something like v1.8.2-0.20220303173753-edfe657b5405, which is older than v0.38.0
+// at the time of this inclusion, v0.38.0 was the latest version available (also tagged as v2.38.0)
+replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.38.0
```

Comment:
We should add this replace to the contrib distro file on opentelemetry-collector-releases